### ISSUE TITLE
Project Thumbnail Background Colors Labels Are Not Descriptive

### ISSUE THREAD

**Issue by Santoz2** [ManifoldScholar/manifold#2432]
When creating a thumbnail for a project, a user has the option to use a default thumbnail with a selected color. However, the labels for these defaults colors do not describe their colors. A user who may see color differently, or not see color at all, may still want to determine what color their readers will see.

## Expected Behavior
Labels should be descriptive and provide useful information. Each thumbnail color should be given a label that is the name of that color. 

## Current Behavior
The labels for project thumbnail colors currently read as "PrimaryBackgroundColor", "SecondaryBackgroundColor", and so on, rather than stating the actual color. 

## Steps to Reproduce
1. Navigate to Admin Interface (via Enter Admin Mode)
2. Navigate to an editable project, landing on the General Attributes page
3. Navigate to the Project Thumbnail section
4. Navigate to the Default section, and use arrow keys to pick colors
5. Note how each color is labelled as "Primary/Secondary/TertiaryBackgroundColor", rather than being given a descriptive name.

## Testing Environment
NVDA 2019.2.1
ESR Firefox 68.2.0 esr

**Comment by zdavis** [comment]
Great catch. Thank you!
